src/core/lib/iomgr/tcp_server_posix.c

```diff
@@ -70,6 +71,8 @@
 
 #define MIN_SAFE_ACCEPT_QUEUE_SIZE 100
 
+int grpc_tcp_server_posix_expand_wildcard_addr = 0;
```

Comment:
ServerBuilder is at the C++ wrapper level. Do we want something at the low-level core C code level?

If it was something we could add at the C++ level, I'd just add it in ChannelArguments.